deepmd/README.md

```diff
@@ -1,23 +1,62 @@
 # DEEPMD
 
-DeePMD (Deep Potential Molecular Dynamics) [1] is a package for easily building performant neural network potentials for carrying out molecular dynamics. The DeePMD code base has been developed such that trained models can be interfaced directly to LAMMPS.
+DeePMD (Deep Potential Molecular Dynamics) [1] is a package for easily building
```

Comment:
I think we need a script that would tie everything up together, starting from cp.x and finishing with the MD run